website/src/components/Submission/FileUpload/SequenceEntryUploadComponent.tsx

```diff
@@ -98,23 +95,9 @@ export const SequenceEntryUpload: FC<SequenceEntryUploadProps> = ({
                 {isMultiSegmented && (
                     <p className='text-gray-400 text-xs mt-3'>
                         {organism.toUpperCase()} has a multi-segmented genome. Please submit one metadata entry with a
-                        unique <i>submissionId</i> for the full multi-segmented sample, e.g. <b>sample1</b>. Sequence
-                        data should be a FASTA file with each header indicating the <i>submissionId</i> and the segment,
-                        i.e.{' '}
-                        {getFirstLightweightSchema(referenceGenomeLightweightSchema).nucleotideSegmentNames.map(
-                            (name, index) => (
-                                <span key={index} className='font-bold'>
-                                    sample1_{name}
-                                    {index !==
-                                    getFirstLightweightSchema(referenceGenomeLightweightSchema).nucleotideSegmentNames
-                                        .length -
-                                        1
-                                        ? ', '
-                                        : ''}
-                                </span>
-                            ),
-                        )}
-                        .
+                        unique <i>submissionId</i> column for the full multi-segmented sample, e.g. <b>sample1</b> and a{' '}
+                        <i>fastaId</i> column with a space-separated list of the fasta headers of all segments, e.g.{' '}
```

Comment:
```suggestion
                        <i>fastaIds</i> column with a space-separated list of the fasta headers of all segments, e.g.{' '}
```
(based on current way of the poll)